src/sagemaker/jumpstart/exceptions.py

```diff
@@ -60,14 +65,13 @@ def get_wildcard_model_version_msg(
 
 def get_proprietary_model_subscription_msg(
     model_id: str,
-    model_version: str,
     subscription_link: str,
 ) -> str:
-    """Returns customer-facing message for using a Marketplace model."""
+    """Returns customer-facing message for using a proprietary model."""
 
     return (
-        f"Using Marketplace model '{model_id}' with version identifier '{model_version}'. "
-        f"Please make sure to subscribe to the model from {subscription_link}"
+        f"INFO: Using proprietary model '{model_id}'. "
```

Comment:
? why not use an `INFO` logger?

Reply:
The idea is to get this message printed out by default using a warning logger, very similar to the wildcard warning message. This is a temp method before we have entitlement check in our control plane, after that we will only log this if the customer is not subscribed to the product.